core/src/main/java/nl/nn/adapterframework/webcontrol/api/TransactionalStorage.java

```diff
@@ -627,11 +627,17 @@ private String getRawMessage(IMessageBrowser<?> messageBrowser, IListener listen
 				} catch (IOException e) {
 					throw new ApiException(e, 500);
 				}
+			} else if(rawmsg instanceof Message){
+				try {
+					msg = ((Message)rawmsg).asString();
+				} catch (IOException e) {
+					throw new ApiException(e, 500);
+				}
```

Comment:
Please add a comment that this code is only here to accommodate for a previous version of the code